app/src/main/java/org/p2p/wallet/svl/model/SendViaLinkClaimingState.kt

```diff
@@ -27,10 +27,18 @@ sealed interface SendViaLinkClaimingState {
         @DrawableRes val iconRes: Int
     ) : SendViaLinkClaimingState {
 
-        constructor() : this(
-            titleRes = R.string.send_via_link_error_failed_parsing_title,
-            subTitleRes = null,
-            iconRes = R.drawable.ic_not_found
-        )
+        companion object {
+            val UNKNOWN_ERROR = ParsingFailed(
+                titleRes = R.string.send_via_link_error_failed_parsing_title,
+                subTitleRes = null,
+                iconRes = R.drawable.ic_not_found
+            )
+
+            val INTERNET_ERROR = ParsingFailed(
```

Comment:
That looks weird. 
Mb better make them as functions? 

Smth like `buildNetworkError(): ParsingFailed`, `buildUnknownError(): ParsingFailed`
What do you think?

Reply:
Will do, thanks!